src/ol/render/canvas.js

```diff
@@ -155,8 +155,8 @@ export const defaultLineWidth = 1;
 
 /**
  * The label cache for text rendering. To change the default cache size of 2048
- * entries, use {@link ol.structs.LRUCache#setSize}.
- * @type {ol.structs.LRUCache.<HTMLCanvasElement>}
+ * entries, use {@link module:ol/structs/LRUCache~LRUCache#setSize}.
```

Comment:
Deep `@link`s like this are the only case where we want to continue using the long module path.